### PR TITLE
Feature/spear bits audit2

### PR DESCRIPTION
**Pull request by invocamanman**
This PR fix the following issues:
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/75
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/71
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/81
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/80
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/79
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/77
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/9
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/10
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/11
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/12
https://github.com/spearbit-audits/review-polygon-zkevm-contracts/issues/13

### CODE REVIEW

**Review by ignasirv — APPROVED** [review]
💯

File: contracts/PolygonZkEVM.sol [hunk 1]
```diff
@@ -427,18 +438,24 @@ contract PolygonZkEVM is OwnableUpgradeable, EmergencyManager {
         bytes32 currentAccInputHash = sequencedBatches[currentBatchSequenced]
             .accInputHash;
 
+        // Store in a temporal variable, for avoid access again the storage slot
+        uint64 orgLastForceBatchSequenced = currentLastForceBatchSequenced;
```
Comment (krlosMata) [inline contracts/PolygonZkEVM.sol:442]: Why `orgLastForceBatchSequenced` naming ?
I think it is better `originLastForceBatchSequenced` or `init...` for readability

```suggestion
        uint64 orgLastForceBatchSequenced = currentLastForceBatchSequenced;
```

**Review by krlosMata — CHANGES_REQUESTED** [review]

File: contracts/PolygonZkEVM.sol (hunk 1 above)
Comment (invocamanman) [reply contracts/PolygonZkEVM.sol:442]: resolved in future following PR ( Since is not an important issue i will pospone it to avoid rebases)

**Review by ignasirv — APPROVED** [review]

**Review by krlosMata — APPROVED** [review]
:smile_cat:

**Pull request merged**